cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -179,6 +179,10 @@
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.10</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
```

Comment:
Better be exclude the transitive one from Kafka. Maven now depends on the depth of the dependency tree to determine the version